models/llama-2-7b-vllm/config.py

```diff
@@ -0,0 +1,148 @@
+import os
+import subprocess
+
+import torch
+from dotenv import load_dotenv
+from transformers import LlamaTokenizer
+
+from src.utils import get_env_var_or_default
+
+load_dotenv()
+
+MODEL_NAME = "llama-2-7b-vllm"
+# INFERENCE CONFIGURATION
+#######################################################################
+# --------------------Notes--------------------------------------------
+# We sometimes implement inference differently for models that have not
+# been trained/fine-tuned vs. those that have been trained/fine-tuned. We refer to the
+# former as "default" and the latter as "trained". Below, you can
+# set your "default inference configuration" and your "trained
+# inference configuration".
+#
+# GENERAL INFERENCE CONFIGURATION
+# -------------------------------
+# This section defines the general inference configuration,
+# which is used for both trained and untrained models.
+# -------------------------------
+
+LOAD_IN_4BIT = False
+TOKENIZER_PATH = f"models/{MODEL_NAME}/model_artifacts/default_inference_weights"
+USE_SYSTEM_PROMPT = False
+USE_EXLLAMA_FOR_UNTRAINED_WEIGHTS = False
+
+
+# DEFAULT INFERENCE CONFIGURATION
+# -------------------------------
+# This section defines the default inference configuration, which may differ from
+# how we implement inference for a trained model.
+# -------------------------------
+
+
+LOCAL_DEFAULT_INFERENCE_WEIGHTS_PATH = f"models/{MODEL_NAME}/model_artifacts/default_inference_weights"
+
+REMOTE_DEFAULT_INFERENCE_WEIGHTS_PATH = "https://pub-df34620a84bb4c0683fae07a260df1ea.r2.dev/Llama-2-7b/"
+
+N_SHARDS = 3
```

Comment:
for 7b, I think this should be set to `2`. I see 2 files in my copy of `llama-2-7b` base weights: `model-00001-of-00002.safetesors` and `model-00002-of-00002.safetesors`

Reply:
Oh, actually I see you have this same variable set below on line 90, perhaps this chunk could be removed?

Reply:
Done!